docs/RaidbossGuide.md

```diff
@@ -155,28 +172,40 @@ Additionally, as with bare `regex` elements, current practice is to use regex re
 Instead, we could use the translations provided in the `timelineReplace` object to do this automagically.
 We're not there yet, but there's always someday.)
 
-**condition: function(data, matches)**
-Activates the trigger if the function returns `true`. If it does not return `true`, nothing is shown/sounded/run. If multiple functions are present on the trigger, this has first priority to run.
+**condition: function(data, matches, output)**
+Activates the trigger if the function returns `true`.
+If it does not return `true`, nothing is shown/sounded/run.
+If multiple functions are present on the trigger, this has first priority to run.
 (Pre-made "canned" conditions are available within [conditions.js](https://github.com/quisquous/cactbot/blob/main/resources/conditions.js).
 Generally speaking it's best to use one of these if it fits the situation.)
 
-**preRun: function(data, matches)**
+**preRun: function(data, matches, output)**
 If the trigger activates, the function will run as the first action after the activation condition is met.
 
-**promise: function(data, matches)**
-If present and a function which returns a promise, will wait for promise to resolve before continuing with trigger. This runs after `preRun` and before the `delaySeconds` delay.
+**promise: function(data, matches, output)**
+If present and a function which returns a promise,
+will wait for promise to resolve before continuing with trigger.
+This runs after `preRun` and before the `delaySeconds` delay.
 
 **delaySeconds**
-An amount of time, in seconds, to wait from the time the regex match is detected until the trigger activates. May be a number or a function(data, matches) that returns a number.
+An amount of time, in seconds, to wait from the time the regex match is detected until the trigger activates.
+May be a number or a `function(data, matches, output)` that returns a number.
```

Comment:
Here and elsewhere, maybe it's worth noting for inexperienced users that `output` is largely meaningless for functions that just return a float or integer?

Reply:
I added some notes to the `output` section above, thanks.  Let me know if you think it needs more.